app/src/main/java/de/tum/in/tumcampusapp/api/TUMCabeClient.java

```diff
@@ -216,6 +219,10 @@ public void deviceUploadGcmToken(DeviceUploadGcmToken verification, Callback<TUM
         service.deviceUploadGcmToken(verification).enqueue(cb);
     }
 
+    public void createMeasurement(WifiMeasurement wifiMeasurement) throws IOException{
+        service.createMeasurement(wifiMeasurement).execute().body();
```

Comment:
Please make this an async function using `enqueue`:
https://futurestud.io/tutorials/retrofit-synchronous-and-asynchronous-requests
https://medium.com/@shelajev/how-to-make-http-calls-on-android-with-retrofit-2-cfc4a67c6254